about.html

```diff
@@ -8,10 +8,10 @@
       <div class="row justify-content-center text-center">
         <div class="col col-lg-10">
           <h2>
-            Real problems and uniquely framed solutions in a real world environment.
+            Real problems meet framed solutions

```

Comment:
This is wrong.

It's about the uniqueness of how solutions are framed and it's about real world environments.

Reply:
Ok, but too many unecessary intensifiers makes for less impact. "Uniquely" and "real-world" need work.